docs/README.md

```diff
@@ -24,6 +24,13 @@ From the `docs` dir, run `./new.sh doc-title`
 
 This will create a new Markdown file for the new doc page, i.e. `docs/1214-doc-title.md`
 
+After executing the `./new/sh` command, make sure to previsualize the new doc with the `make web` command in the root directory. This will trigger `docusaurus start`, [creating a local dev server](https://docusaurus.io/docs/cli#docusaurus-start-sitedir).  
+`make web` **is required** when creating a new doc because it exports a new `\_redirects` file, located in `/website/static/\_redirects`. This is a configuration file that maps the docs ID with their respective filename, so every website can be also reached in the docs with the ID only.  
+Try it out and you'll see how `https://docs.dagger.io/1247` redirects to `https://docs.dagger.io/1247/dagger-fs`  
+Don't worry if the redirection doesn't work in your new doc, as it's a server-side implementation that will only take effect in production. Just make sure it has the same format as the rest of the mappings.
+
+Once created and previsualized, run `npx docusaurus build`. This command verifies no links are broken when parsing markdown, among other things, so it's a good way to "test" your new doc.
```

Comment:
seems you need to run `npx docusaurus build` from the `website/` directory, right? While you run `make web` from the root of the repo.